### PR TITLE
Chat fe

### PR DESCRIPTION
**Pull request by TarasKun**
Add front end part of chat.

### CODE REVIEW

**Review by DmytroDidukh — APPROVED** [review]
Great job!

File: src/components/chat/chat.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+export const Chat = () => {
+  const [visible, setBlock] = useState(false);
+  const [mailFormVisible, setMailFormVisible] = useState(false);
+  const { language, themeMode } = useSelector((state) => ({
+    language: state.Language.language,
+    themeMode: state.Theme.lightMode
+  }));
+  const style = useStyles({ themeMode, visible, mailFormVisible });
+
+  const activeMessenger = () => (
```
Comment (Alan18081) [inline src/components/chat/chat.js:22]: Please, move this component to the separate file, it definitely looks like a separate component

File: src/components/chat/chat.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+              {CHAT[language].ourContacts}
+            </span>
+            <span className={style.phoneNumbers}>
+              <img className={style.logo} src={kyivstar} alt='kyivstar' />{' '}
+              +38-(067)-777-55-33
+            </span>
+            <span className={style.phoneNumbers}>
+              <img className={style.logo} src={vodafone} alt='vodafone' />
+              {'  '}
+              +38-(050)-999-77-66
```
Comment (Alan18081) [inline src/components/chat/chat.js:92]: This number should be taken from the database and should not be hardcoded

File: src/components/chat/chat.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+          >
+            <CancelIcon />
+          </div>
+          <div className={style.contacts}>
+            <span className={style.contactsTitle}>
+              {CHAT[language].ourContacts}
+            </span>
+            <span className={style.phoneNumbers}>
+              <img className={style.logo} src={kyivstar} alt='kyivstar' />{' '}
+              +38-(067)-777-55-33
```
Comment (Alan18081) [inline src/components/chat/chat.js:87]: This number should be taken from the database and should not be hardcoded

File: src/components/chat/chat.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        multiline
+        rows={4}
+      />
+      <Button className={style.btnSend}>{CHAT[language].sendBtn}</Button>
+    </div>
+  );
+
+  return (
+    <div>
+      <div className={visible ? style.iconsMessengers : style.disable}>
```
Comment (Alan18081) [inline src/components/chat/chat.js:58]: Ideally, when it is not visible it should not be rendered as a component, it is better to use conditional rendering

File: src/components/chat/chat.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        <div>
+          <MessengerCustomerChat
+            pageId='106973281128068'
```
Comment (Alan18081) [inline src/components/chat/chat.js:61]: Please, make the `pageId` and `appId` environment variables  with the same approach as `REACT_APP_API_URL` is implemented

File: src/components/chat/chat.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+            appId='768636160372711'
+          />
+        </div>
+        <div className={mailFormVisible ? style.msgIconActive : style.msgIcon}>
+          <MailOutlineIcon
+            className={style.icon}
+            onClick={() => setMailFormVisible(!mailFormVisible)}
+          />
+        </div>
+        <div className={mailFormVisible ? style.mailForm : style.disable}>
```
Comment (Alan18081) [inline src/components/chat/chat.js:71]: Ideally, when it is not visible it should not be rendered as a component, it is better to use conditional rendering

File: src/components/chat/chat.js (hunk 1 above)
Comment (TarasKun) [reply src/components/chat/chat.js:22]: done

File: src/components/chat/chat.js (hunk 5 above)
Comment (TarasKun) [reply src/components/chat/chat.js:61]: done

File: src/components/chat/chat.js (hunk 4 above)
Comment (TarasKun) [reply src/components/chat/chat.js]: fixed

File: src/components/chat/chat.js (hunk 6 above)
Comment (TarasKun) [reply src/components/chat/chat.js]: fixed

File: src/components/chat/chat.js (hunk 2 above)
Comment (TarasKun) [reply src/components/chat/chat.js]: fixed

File: src/components/chat/chat.js (hunk 3 above)
Comment (TarasKun) [reply src/components/chat/chat.js]: fixed

File: src/components/chat/chat.js [hunk 7]
```diff
@@ -4,11 +4,10 @@ import CancelIcon from '@material-ui/icons/Cancel';
...
-import { Button, TextField } from '@material-ui/core';
+import { FOOTER_CONTACTS } from '../../translations/footer.translations';
+import { ActiveMessenger } from './activeMessenger';
 import { useStyles } from './chat.style';
 import { CHAT } from '../../translations/chat.translation';
-import kyivstar from '../../images/kyivstar.png';
-import vodafone from '../../images/vodafone.png';
 
 export const Chat = () => {
   const [visible, setBlock] = useState(false);
```
Comment (Alan18081) [inline src/components/chat/chat.js:13]: What does `visible` reflect? As I understand it allows to hide all icons, doesn't it?

File: src/components/chat/activeMessenger.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+export const ActiveMessenger = ({
+  themeMode,
+  visible,
+  mailFormVisible,
+  language
+}) => {
+  const style = useStyles({ themeMode, visible, mailFormVisible });
+
+  return (
+    <div className={style.formFieldActive}>
```
[the remark this answers is not in the document]
Comment (TarasKun) [reply src/components/chat/activeMessenger.js:15]: Done

File: src/components/chat/chat.js (hunk 7 above)
Comment (TarasKun) [reply src/components/chat/chat.js:13]: Yes, for a better understanding of what this variable does, I passed the variable 'visible' to 'iconsVisible'

File: src/components/chat/activeMessenger.js [hunk 9]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (TarasKun) [reply src/components/chat/activeMessenger.js:1]: Done

File: nginx.conf [hunk 10]
```diff
@@ -2,9 +2,9 @@ server {
     location / {
 	add_header 'Access-Control-Allow-Origin' '*';
         root /usr/share/nginx/html;
-        index index.html;
+        index.js index.js.html;
```
Comment (Alan18081) [inline nginx.conf:5]: Why have you made this change?

File: README.md [hunk 11]
```diff
@@ -1,267 +0,0 @@
-# Horondi
```
Comment (Alan18081) [inline README.md:1]: Why have you removed the `README` file, please, revert this changes

File: src/components/chat/chat.js [hunk 12]
```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
```
Comment (Alan18081) [inline src/components/chat/chat.js:1]: This component should be inside `containers` folder because it is connected to the redux. Also, it is better to move `ActiveMessenger` inside the folder of `chat` component because it is going to be used only there, in `chat`

File: README.md (hunk 11 above)
Comment (TarasKun) [reply README.md:1]: returned the file README, it was mistake

File: src/components/chat/chat.js (hunk 12 above)
Comment (TarasKun) [reply src/components/chat/chat.js:1]: now component active-messenger inside folder chat, and component chat inside containers

File: nginx.conf (hunk 10 above)
Comment (TarasKun) [reply nginx.conf]: fixed
